metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -74,72 +74,6 @@ def get_system_auth() -> Optional[str]:
     return None
 
 
-def get_url_and_token():
-    gms_host_env, gms_token_env = get_details_from_env()
-    if len(config_override.keys()) > 0:
-        gms_host = config_override.get(ENV_METADATA_HOST_URL)
-        gms_token = config_override.get(ENV_METADATA_TOKEN)
-    elif config_utils.should_skip_config():
-        gms_host = gms_host_env
-        gms_token = gms_token_env
-    else:
-        config_utils.ensure_datahub_config()
-        gms_host_conf, gms_token_conf = config_utils.get_details_from_config()
-        gms_host = first_non_null([gms_host_env, gms_host_conf])
-        gms_token = first_non_null([gms_token_env, gms_token_conf])
-    return gms_host, gms_token
-
-
-def get_token():
-    return get_url_and_token()[1]
-
-
-def get_session_and_host():
```

Comment:
to reduce the amount of code changes, you could make this call get_default_graph() and then return the tuple

basically keep it as a compatibility shim

Reply:
Doing that causes a cyclical import dependency. It was my first original implementation.

Reply:
you can put the import inside the `get_session_and_host` method to avoid that issue